<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomizePageLineWidget</class>
 <widget class="QWidget" name="CustomizePageLineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Line</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="Line_lineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="Alligment_comboBox">
     <item>
      <property name="text">
       <string>Left</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Center</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Right</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Font_pushButton">
     <property name="text">
      <string>Font</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="textScrolling">
     <property name="text">
      <string>Text scrolling</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
